builtin/providers/aws/resource_aws_sqs_queue.go

```diff
@@ -177,6 +178,9 @@ func resourceAwsSqsQueueRead(d *schema.ResourceData, meta interface{}) error {
 					}
 					d.Set(iKey, value)
 				} else {
+					if iKey == "policy" {
+						*attrmap[oKey] = normalizeJson(*attrmap[oKey])
+					}
```

Comment:
SQS Policy is always change every plan in current version.
If need split to other pull-request, I will do it.

```
resource "aws_s3_bucket" "bucket" {
   bucket = "tf-test-bucket-123456789" 
}

resource "aws_sqs_queue" "queue" {
    name = "terraform-test-queue"
        policy = <<POLICY
{
  "Version": "2012-10-17",
  "Statement": [
    {
      "Effect": "Allow",
      "Principal": "*",
      "Action": "sqs:SendMessage",
          "Resource": "arn:aws:sqs:*:*:terraform-test-queue",
      "Condition": {
          "ArnEquals": { "aws:SourceArn": "${aws_s3_bucket.bucket.arn}" }
      }
    }
  ]
}
POLICY
}
```

```
$ terraform plan
...
~ aws_sqs_queue.queue
    policy: "{\"Version\":\"2012-10-17\",\"Statement\":[{\"Effect\":\"Allow\",\"Principal\":\"*\",\"Action\":\"sqs:SendMessage\",\"Resource\":\"arn:aws:sqs:*:*:terraform-test-queue\",\"Condition\":{\"ArnEquals\":{\"aws:SourceArn\":\"arn:aws:s3:::tf-test-bucket-123456789\"}}}]}" => "{\n  \"Version\": \"2012-10-17\",\n  \"Statement\": [\n    {\n      \"Effect\": \"Allow\",\n      \"Principal\": \"*\",\n      \"Action\": \"sqs:SendMessage\",\n\t  \"Resource\": \"arn:aws:sqs:*:*:terraform-test-queue\",\n      \"Condition\": {\n          \"ArnEquals\": { \"aws:SourceArn\": \"arn:aws:s3:::tf-test-bucket-123456789\" }\n      }\n    }\n  ]\n}\n"
...
```